topdown/eval.go

```diff
@@ -664,18 +664,12 @@ func (e *eval) evalNotPartialSupport(negationID uint64, expr *ast.Expr, unknowns
 	}
 
 	// Save expression that refers to support rule set.
-
-	terms := expr.Terms
-	expr.Terms = nil // Prevent unnecessary copying the terms.
-	cpy := expr.Copy()
-	expr.Terms = terms
+	cpy := copyExprSansTerms(expr)
```

Comment:
Is this change related to the payload suppress change ?

Reply:
🙈 this is where it went. I had been looking for that. It's #4503. Will fix this PR.